src/main/java/com/softserve/academy/studhub/entity/FeedBack.java

```diff
@@ -0,0 +1,42 @@
+package com.softserve.academy.studhub.entity;
+
+import lombok.EqualsAndHashCode;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+@EqualsAndHashCode
+@Table(name = "feedback")
+public class FeedBack {
+
```

Comment:
Please change the class name to 'Feedback' and table name to 'feedbacks'

Reply:
Done)